python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -508,7 +539,7 @@ async def poll_loop(self):
                 return
 
 
-def _statusBits(fields, value):
+def _status_bits(fields, value):
     """Helper function. Converts value bits into boolean fields.
 
     Parameters
```

Comment:
In line 516 bellow. 

If `fields` is a list of strings, the type should be `list` [ `str` ]


In line 556, the type should be `dict` [ `str`, `bool` ]